spec/unit/provider/remote_directory_spec.rb

```diff
@@ -79,7 +79,7 @@ class Chef::CFCCheck
     end
 
     it "configures access control on intermediate directorys" do
-      directory_resource = @provider.send(:resource_for_directory, File.join(Dir.tmpdir, "intermediate_dir"))
+      directory_resource = @provider.send(:directory_resource, File.join(Dir.tmpdir, "intermediate_dir"))
```

Comment:
Probably a little cleaner to have this be `@api private` but ruby-public. nbd though.